string SWITCH_TO_ACTION = switch-to-action
string SWITCH_TO_ACTION_BY_ID = switch-to-action-by-id
string SELECT_ACTION_STEP = select-action-step
string command
string param
